code/client/src/core/ui/world_debug.cpp

```diff
@@ -11,12 +11,16 @@
 #include "sdk/mafia/framework/c_mafia_dbs.h"
 #include "sdk/ue/game/traffic/c_streaming_traffic_module.h"
 #include "sdk/ue/gfx/environmenteffects/c_gfx_environment_effects.h"
+#include "sdk/mafia/database/c_ui_database.h"
+#include "sdk/mafia/ui/hud/race_xbin.h"
+#include "sdk/mafia/ui/hud/c_hud_controller.h"
 
 #include "sdk/entities/c_car.h"
 #include "sdk/entities/c_vehicle.h"
 
 #include "game/helpers/controls.h"
 #include "game/helpers/human.h"
+#include "../../sdk/mafia/ui/c_game_gui_2_module.h"
```

Comment:
You can use direct "sdk" magic path here

Reply:
and group sdk import together